src/content/basics/nodepools/index.md

```diff
@@ -1,29 +1,34 @@
 ---
 title: Node Pools
 description: A general description of node pools as a concept, it's benefits, and some details you should be aware of.
-date: 2019-11-07
+date: 2019-11-18
 weight: 15
 type: page
 categories: ["basics"]
 ---
 
-# Node Pools
+# Node Pools on AWS
 
 <div class="well disclaimer">
 Node pools are a new concept to be introduced soon to Giant Swarm customers on AWS.
 </div>
 
+## Definition
+
+A node pool is a group of nodes within a cluster that all have the same configuration. Node pools use a NodeConfig specification. Each
```

Comment:
I never heard of that `NodeConfig`

Reply:
I actually copied the definition from Google. I knew you would set me straight in whether this was a general concept or a GCP one. I'll get rid of it. Thanks for the feedback.